Docs: update DVC Use Case (#132)
iterative/mlem.ai: content/docs/use-cases/dvc.md

```

> Next let's create an isolated virtual environment to cleanly install all the

Reviewer: ```suggestion
Next let's create a Python virtual environment to cleanly install all the
```

> requirements (including MLEM) there.
> 
> This will also install DVC with `pip`. Learn about other ways to install DVC
> [here](https://dvc.org/doc/install).

Reviewer: ```suggestion
requirements with `pip` (including DVC and MLEM).

> Learn about other ways to install DVC [here](https://dvc.org/doc/install).
```

Reviewer: Why do we want to mention that there are other ways to install DVC? Seems irrelevant for this use case.

Author: Agreed. Let's delete that mention then.

> $ pip install -r requirements.txt

Reviewer 2: minor: something is wrong with indent?

Author: Sorry, what do you mean? I see no problems
<img width="273" alt="image" src="https://user-images.githubusercontent.com/6797716/178495642-d46103fb-964a-4504-b3bd-9aecf23e6c95.png">

Reviewer 2: please disregard, my bad

```

> This will install DVC with `pip`. Learn about other ways to install DVC

Reviewer 2: Why not mention DVC on the line 17? (MLEM and DVC) ...

`This command also installs DVC ... it's not required to have DVC as a dependency, this is done for simplicity ... there are other options to install DVC` - something along these lines could be a bit better, wdyt?

Author: 1. Moved that above as you suggested.
2. `it's not required to have DVC as a dependency, this is done for simplicity` -- not true, since to use MLEM models you'll need to have DVC installed: model/data binaries are uploaded to cloud here.

Author: Ah, you right indeed! Checked the implementation, and MLEM uses DVC API. Let's add `mlem[dvc, dvc-s3]` then.

Author: https://github.com/iterative/mlem/pull/343

Author: Ok, now you could do `pip install mlem[dvc, dvc-s3]`. Should we mention now the other ways to install DVC? (I also need to update MLEM generation code in `example-repos-dev` to have this in `requirements.txt`). @shcheklein

Author: Ok, @jorgeorpinel suggested below to remove this mention. I agree, so I removed it. I think we can merge this now. Please approve if you agree.

Author: Sorry, let me update this PR first. Posted a comment below.
